tests/phpunit/CRM/Activity/Form/Task/PDFLetterCommonTest.php

```diff
@@ -140,15 +140,16 @@ public function testCreateDocumentSpecialTokens(): void {
   }
 
   /**
+   * Unknown tokens are removed at the very end.
+   *
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
   public function testCreateDocumentUnknownTokens(): void {
     $activity = $this->activityCreate();
-    $html_message = 'Unknown token: {activity.something_unknown}';
+    $html_message = 'Unknown token: ';
```

Comment:
What is this testing now without an unknown token in the input message? How about:
```
$html_message = 'Unknown token: {activity.something_unknown}';
$expected_output = 'Unknown token: ';
...
$this->assertEquals($expected_output, $output[0]);
```

Reply:
Oh dang - did I remove from the wrong line!

Reply:
https://github.com/civicrm/civicrm-core/pull/21585 puts it back